app.js

```diff
@@ -12,12 +12,41 @@ app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.static(path.join(__dirname, 'public')));
 
+
+
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beers => res.render('beers', { beers }))
+
+    .catch(error => console.log(error));
+});
+
+app.get('/randomBeer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(randomBeer => res.render('randomBeer', { randomBeer }))
+      
+
+    .catch(error => console.log(error));
+
+});
+
+
+app.get('/beers/:id', (req, res) => {
+  punkAPI
+    .getBeers(req.params.id)
```

Comment:
The `punkAPI` method to request just one specific beer through the id is `.getBeer()`